HeavyIonsAnalysis/JetAnalysis/src/HiInclusiveJetAnalyzer.cc

```diff
@@ -247,6 +250,13 @@ HiInclusiveJetAnalyzer::beginJob() {
   t->Branch("jtm",jets_.jtm,"jtm[nref]/F");
   t->Branch("jtarea",jets_.jtarea,"jtarea[nref]/F");
 
+  //for reWTA reclustering
+  // disable for calo jets
+  if(doWTARecluster_ && !(jetName_.find("Calo") != std::string::npos)){
```

Comment:
Why do we disable this for calo jets?  It's an unusual use case but personally, I would rather leave the option there -- is there a forest coding guideline on these kinds of design choice?

Reply:
I needed to disable calo jets, because the test for pp and pp MC data samples failed for calo jets. The error was RefCore: A request to resolve a reference to a product of type 'reco::Candidate' with ProductID '2:55' can not be satisfied because the product cannot be found.

Reply:
I see - ok, in this case, let's leave it like this.  If people need WTA on calo jets for whatever reason we do another code update